.github/workflows/lint.pr.yml

```diff
@@ -0,0 +1,18 @@
+name: Lint
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
```

Comment:
Another minor comment. Would it make sense to set names for those actions?
E.g.    
```
- name: Checkout
  uses: actions/checkout@v3
...
- name: Install Node.js
  uses: actions/setup-node@v3
...
- name: Install pnpm
  uses: pnpm/action-setup@v2

Reply:
Nah, it gives them clear enough names, and those things will very rarely fail.

Reply:
Based on what comes after “uses:” I actually had to do some research to understand what’s happening, because I’m not yet that familiar with GitHub actions. I thought others could have a similar problem in the future but maybe that’s a good learning point.